src/main/java/Sb2.java

```diff
@@ -23,6 +23,8 @@
     private Sprites sprites;
     private String name;
     private int globalVariables;
```

Comment:
Also, I don't think we use [this](https://github.com/JaysGitLab/cs-5666-scatt-jacc-caj/pull/47/files/7a688216c6fa53e22c710c97197b987c0500def9..f8abe8de80741ae859753a1472ad6dde0ca60dcd#diff-704fc1a2160808ba6799856e052ff759L13) either